flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSink.java

```diff
@@ -258,4 +262,58 @@ public void testShuffleByPartitionWithSchema() throws Exception {
       Assert.assertEquals("There should be only 1 data file in partition 'ccc'", 1, partitionFiles("ccc"));
     }
   }
+
+  @Test
+  public void testTwoSinksInDisjointedDAG() throws Exception {
+    Map<String, String> props = ImmutableMap.of(TableProperties.DEFAULT_FILE_FORMAT, format.name());
+
+    final String leftTablePath = TEMPORARY_FOLDER.newFolder().getAbsolutePath().concat("/left");
```

Comment:
Unit tests looks good to me.  One minor thing:  in apache iceberg codebase,   I have almost never seen a situation where local variables are declared as final, @rdblue , Is this an implicit code style rule?

Reply:
done. I removed all the final modifiers for local vars in this PR